scripts/windows/test/Setup-Env.ps1

```diff
@@ -87,6 +87,7 @@ If ($osEdition -eq "IoTUAP")    # Windows IoT Core - update iotedge
             if($residualModules.Length -gt 0) {
                 docker -H npipe:////./pipe/iotedge_moby_engine rm -f $residualModules
             }
+	    docker -H npipe:////./pipe/iotedge_moby_engine system prune -a -f
```

Comment:
```suggestion
	    
	      docker -H npipe:////./pipe/iotedge_moby_engine system prune -a -f
```

Reply:
just add spacing and add a new line after if-block.

Reply:
Done.